modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -24,12 +23,7 @@ Hot plug a secondary network interface to a virtual machine (VM) while the VM is
 $ virtctl start <vm_name> -n <namespace>
 ----
 
-. Use the following command to add the new network interface to the running VM. Editing the VM specification adds the new network interface to the VM and virtual machine instance (VMI) configuration but does not attach it to the running VM.
-+
-[source,terminal]
-----
-$ oc edit vm <vm_name>
-----
+. Use your preferred text editor to edit the `VirtualMachine` manifest and add the new network interface to the running VM. Editing the VM specification adds the new network interface to the VM and virtual machine instance (VMI) configuration but does not attach it to the running VM.
```

Comment:
> Editing the VM specification adds the new network interface to the VM and virtual machine instance (VMI) configuration but does not attach it to the running VM.

I don't follow this part, could you please elaborate?

Reply:
Thank you for the clarification.
How about emphasizing that the actual attachment will happen after the migration?
```suggestion
. Use your preferred text editor to edit the `VirtualMachine` manifest and add the new network interface to the running VM. Editing the VM specification adds the new network interface to the VM and virtual machine instance (VMI) specification. The new interface will be hot-plugged upon completion of the automatically initiated migration.
 ```